src/nupic/algorithms/Anomaly.cpp

```diff
@@ -109,10 +108,17 @@ Real32 Anomaly::compute(
   {
     score = (score >= binaryThreshold_) ? 1.0 : 0.0;
   }
+  // round result (to avoid near zero rounding errors)
+  score = roundf(score*(1/PRECISION))/PRECISION;
```

Comment:
@cogmission I think that's a cannon for a sparrow (do you say that?:P) I don't want big precision, the opposite. And introducing a library just to round a number to 4th decimal digit, in context with speaking about optimizations in the other issue...well :confused: